app/views/layouts/base.html.erb

```diff
@@ -30,6 +30,8 @@
       <% end %>
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app", "data-turbolinks-track" => true %>
+    <script type="text/javascript" src="/webpack/webcomponents-bundle.js" data-turbolinks-track="true"></script>
```

Comment:
Can we use a helper for this path? In production this doesn't work. AFAIK there's it's `/public/webpack/...`. Also, we send infinite caching headers in that directory so you'd never be able to update it.

Reply:
Yeah, sorry not to comment beforehand, I agreed with @sharvit he will take a look on those constructs (webpack copying) and will suggest improvements. I believe we should somehow force it into the webpack manifest without compiling it. Or the other solution was to skip webpack altogether and copy it through pipeline, but I believe we want to remove pipeline assets.

Reply:
What if you add it to the `webpack.vendor.js` file? Will it replace all the changes here and in `webpack.config.js`?

Reply:
Works nice and smooth for dev and production :man_shrugging:  I smell a black magic :eyes:
:dart: Thanks @sharvit :+1:

Reply:
Not really a black magic, it's actually just a webpack entry:
https://github.com/theforeman/foreman/blob/develop/config/webpack.config.js#L97

There is a WIP of managing with NPM, see:
https://github.com/sharvit/foreman-js